src/mui-setup.js

```diff
@@ -19,14 +18,15 @@ export default function muiSetup(direction) {
       boxShadow: '0 0 0 2px #3f8ab3 inset',
       outline: 'none',
     };
+    muiConfig.overrides.MuiToolbar = {
+      regular: {
+        minHeight: '49px !important',
+        paddingRight: '24px !important;',
+      },
+    };
+    muiConfig.overrides.MuiTableContainer.root.borderBottom = '0';
   }
 
-  const theme = createTheme(adaptV4Theme({ ...muiConfig, direction }));
-  const generateClassName = createGenerateClassName({
```

Comment:
I think we want to keep this. This makes the classes unique, to make sure no weird stuff happens when you have multiple instances of a table. Or is this not applicable when switching to styled/sx

Reply:
this is not applicable when switching to styled/sx

Reply:
https://mui.com/guides/classname-generator/ there is something new related to classname in v5

Reply:
So I guess we should use that instead.

Reply:
Done

Reply:
Reverted. As it breaks the style of MuiTableSortLabel. 
And I found that 
`&:hover": {
            "& icon": {
              "opacity": 0.5
            }
          },
          "&.Mui-active": {
            "&& icon": {
              "opacity": 1
            }
          }` in MuiTableSortLabel does not work as expected.